ci/src/cI_web_templates.ml

```diff
@@ -51,9 +51,9 @@ let gh_target_url = function
   | `Ref r ->
     let { CI_projectID.user; project } = CI_github_hooks.Ref.project r in
     let id = CI_github_hooks.Ref.name r in
-    match Datakit_path.unwrap id with
-    | "tags" :: id -> Fmt.strf "https://github.com/%s/%s/releases/tag/%s" user project (String.concat ~sep:"/" id)
-    | _            -> Fmt.strf "https://github.com/%s/%s/tree/%a" user project Datakit_path.pp id
+    match String.cut ~sep:"tags/" id with
```

Comment:
(after some discussion)

- Using strings isn't great, because ref names do have structure and it's useful to match on it.

- `Datakit_path` could expose the type (`type t = private string list`). I didn't do this because I was considering making a more optimal representation (e.g. that makes appending fast), but maybe having to call `unwrap` is too annoying.

- We could abstract this with another type "git ref name" and let some library convert between this type and DataKit paths.